charts/argo-cd/values.yaml

```
@@ -279,6 +279,10 @@ configs:
     # -- Enables use of the Progressive Syncs capability
     applicationsetcontroller.enable.progressive.syncs: false
 
+    # When specifying multiple namespaces, specify them as a comma-separated string
+    # e.g., "namespace1, namespace2"
```

- we can align upstream's comment, [here](https://github.com/argoproj/argo-cd/blob/v2.12.3/docs/operator-manual/argocd-cmd-params-cm.yaml#L205-L206)
- In order to show README.md properly, need annotation
```suggestion
+    # -- A list of glob patterns specifying where to look for ApplicationSet resources. (e.g. `"namespace1, namespace2"`)
```

---

Thank you for the information!